mkFit/MkBuilder.cc

```diff
@@ -1532,7 +1534,12 @@ void MkBuilder::PrepareSeeds()
   //0 = not set; 1 = high pT central seeds; 2 = low pT endcap seeds; 3 = all other seeds
   assign_seedtype_forranking();
 
-  fit_seeds();
+  // Do not refit cmssw seeds (this if was nested in fit_one_seed_set() until now).
+  // Eventually we can add force-refit option.
+  if (Config::seedInput != cmsswSeeds)
+  {
+    fit_seeds();
+  }
```

Comment:
I'm curious if reverting this will fix the efficiency? the old code was pushing things in and out, unclear if there was something useful there that's not done already in (what remains now) just the import_seeds